### PR TITLE
Adds App Splash Screen

### PR DESCRIPTION
**Pull request by redreceipt**
## DESCRIPTION

splash

### What does this PR do, or why is it needed?

Adds launch screen of onboarding navigation

### What design trade-offs/decisions were made?

There's got to be a better way but I couldn't figure it out. Currently, I'm matching the cover image to the `Dimensions` width and height, which skews the aspect ratio. Not a huge issue if they're close but could potentially look really bad on tablets.

### How do I test this PR?

`yarn storybook` and `yarn test`

### What automated tests did you write?

The same ones that Burke wrote for `AskName`

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [#563]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by redreceipt** [comment]
@burkeshartsis is there a way I can get the window size from `ui-kit` or something? I'm using `resizeMode: "cover"` but it seems that's not going to suffice.

screen shot 2019-03-04 at 11 45 36 am

**Comment by burkeshartsis** [comment]
If you can't flex it to fill and use `cover` try absolute positioning.

**Comment by redreceipt** [comment]
`absolute` with `top: 0` positions it about 50 px from the top so there's something about the `<Slide>` container I don't understand.

**Comment by burkeshartsis** [comment]
I probably shouldn't have forced the use of `PaddedView` [here](https://github.com/ApollosProject/apollos-prototype/blob/552-onboarding-notifications-screen/packages/apolloschurchapp/src/ui/Onboarding/Slide/index.js#L49). I actually was just in the process of removing it in my current PR so that padding would be controlled by the `children` rather than `Slide`. I can pull the changes into a separate PR if that's helpful or you can test by removing it and wait for my screen to be merged.

Some additional thinking on `Slide` would suggest that `children` maybe shouldn't be wrapped in anything. I'm actually running into positioning issues because of the current flex setup. This could potentially be changed to eliminate that need. I should probably look at adding a `ScrollView` to `Slide` as well so that the layout doesn't break on small screens.

**Comment by redreceipt** [comment]
@burkeshartsis is this screen outside the `Slide` or is it the first element in the list? We probably don't need to be able to go back to it.

**Comment by burkeshartsis** [comment]
It's outside of onboarding all together. It's the app splash screen (the first thing someone sees after the loading screen/launch screen). Assuming the user went through onboarding then logged out of the app it would kick them to this screen. If they they are logged they will always go directly to the home tab.

**Comment by redreceipt** [comment]
ah ok, so we need to tie it into the first `Slide` screen with a "Let's Go" button. Not 100% sure how to do that with the `ui-kit`.

**Comment by burkeshartsis** [comment]
Nope. So it can be built using `Slide` but it won't be part of the `Onboarding` sequence. I'll dig into it when I review the PR.

**Comment by burkeshartsis** [comment]
Also if you don't mind applying some of the code clean up recommendations from this PR to your other ones that would keep me from flooding your PR's with minor code suggestions 😉

File: packages/apolloschurchapp/src/ui/Onboarding/slides/Launch/index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+const CoverImage = styled(() => ({
+  position: 'absolute',
+  marginTop: -50,
+  width: Dimensions.get('window').width + 20,
+  height: Dimensions.get('window').height + 20,
+}))(Image);
+
+const Launch = memo(({ slideTitle, description, ...props }) => (
+  <Slide {...props}>
+    <CoverImage source={require('./launch.jpg')} />
```
[the remark this answers is not in the document]
Comment (burkeshartsis) [reply packages/apolloschurchapp/src/ui/Onboarding/slides/Launch/index.js]: also lets make sure we are supporting @2x and @3x screen densities. Super easy to do. Documentation is [here](https://facebook.github.io/react-native/docs/images)

[the remark this answers is not in the document]
Comment (redreceipt) [reply packages/apolloschurchapp/src/ui/Onboarding/slides/Launch/index.js]: React Native recommends using static image urls (https://stackoverflow.com/a/36677492/1777570) and you can only pass in string literals (https://stackoverflow.com/questions/47887697/require-must-have-a-single-string-literal-argument-react-native)

**Comment by redreceipt** [comment]
@burkeshartsis ok got the splash screen PR cleaned, still have two unresolved comments:
1. `require` needs a string literal and `uri: <URI>` isn't recommended apparently so a little unsure how to make image URI dynamic.
2. definitely need a little help trying to get the image to "cover" the screen without hardcoded values. I feel like I tried everything.

**Pull request merged**